packages/tests/unit-tests/test-cases/expected/vue/refs-as-attribute.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+  <div
+    ><HelperWidget
+      :forwardRef="this.$refs.forwardRef"
+      :someRef="this.$refs.someRef"
+      :refProp="
+        (props === undefined || props === null ? undefined : props.refProp) &&
```

Comment:
Right now I haven't figure out how to shorten this construction, maybe we need to add simple card to fix it, but it's valid